Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift

```diff
@@ -59,13 +60,13 @@ class TestListViewController: UIViewController {
         //If a test run has just updated before this view appeared
         if displayTestRunAlert {
             //Alert the user that the update was successful
-            let updateSuccededAlert = UIAlertController(title: "Run updated", message: "Test run was successfully committed.", preferredStyle: UIAlertControllerStyle.alert)
+            let updateSucceededAlert = UIAlertController(title: updatedRunName + " updated", message: updatedRunName + " was successfully committed.", preferredStyle: UIAlertControllerStyle.alert)
```

Comment:
I am not sure if we want the run name in the title and body, just the title or just the body. I went with both but I am not 100% sure that is the right way to go. Especially when there are long run names.